packages/vue-client/src/views/HomeView.vue

```diff
@@ -1,11 +1,12 @@
 <script setup lang="ts">
 import GameCreationForm from "@/components/GameCreation/GameCreationForm.vue";
 import GameList from "@/components/GameList.vue";
+import { SITE_NAME } from "@ogfcommunity/variants-shared";
 </script>
 
 <template>
   <main>
-    <h2>Welcome to Go variants!</h2>
+    <h2>Welcome to {{ SITE_NAME }}!</h2>
```

Comment:
This was mainly for testing.  I can remove it before merging, but want to make sure we landed on an agreeable configuration first.